src/test/java/edu/ksu/canvas/attendance/form/CourseConfigurationValidatorUTest.java

```diff
@@ -57,11 +57,11 @@ public void validate_rejectMinutesPerSessionGreaterThanTotalMinutes() {
     public void validate_rejectWrongPoints() {
         courseConfigurationForm.setAssignmentName("Attendance Assignment");
         courseConfigurationForm.setGradingOn(true);
-        courseConfigurationForm.setAssignmentPoints(100.0);
-        courseConfigurationForm.setPresentPoints(100.0);
-        courseConfigurationForm.setAbsentPoints(120.0);
-        courseConfigurationForm.setExcusedPoints(0.0);
-        courseConfigurationForm.setTardyPoints(0.0);
+        courseConfigurationForm.setAssignmentPoints("100.0");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "100.0" 9 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)